Changelog.md

```diff
@@ -5,143 +5,143 @@
+    * [BitBucket pull request 208](https://osrf-migration.github.io/ignition-gh-pages/#!/ignitionrobotics/ign-common/pull-requests/208)
 
 1. `SystemPaths::FindSharedLibrary`: return input if its is an absolute path to a file
-    * [Pull request 206](https://bitbucket.org/ignitionrobotics/ign-common/pull-request/206)
+    * [BitBucket pull request 206](https://osrf-migration.github.io/ignition-gh-pages/#!/ignitionrobotics/ign-common/pull-requests/206)
 
 1. Fix `Image::ConvertPixelFormat` for `BAYER_BGGR8`
-    * [Pull request 211](https://bitbucket.org/ignitionrobotics/ign-common/pull-requests/211)
+    * [BitBucket pull request 211](https://osrf-migration.github.io/ignition-gh-pages/#!/ignitionrobotics/ign-common/pull-requests/211)
 
 1. Support custom callbacks to find files on global interface
-    * [Pull request 226](https://bitbucket.org/ignitionrobotics/ign-common/pull-request/226)
+    * [BitBucket pull request 226](https://osrf-migration.github.io/ignition-gh-pages/#!/ignitionrobotics/ign-common/pull-requests/226)
 
 ## Ignition Common 3.3.0 (2019-08-27)
 
 1. skip GTSMeshUtils test on Windows due to issue #50
-    * [Pull request 205](https://bitbucket.org/osrf/gazebo/pull-request/205)
+    * [BitBucket pull request 205](https://osrf-migration.github.io/gazebo-gh-pages/#!/osrf/gazebo/pull-requests/205)
 
 1. Add bvh animation support for skeleton and TrajectoryInfo
-    * [Pull request 197](https://bitbucket.org/osrf/gazebo/pull-request/197)
+    * [BitBucket pull request 197](https://osrf-migration.github.io/gazebo-gh-pages/#!/osrf/gazebo/pull-requests/197)
 
 1. ColladaLoader: fix wrong node weights caused by buffer overflow bug
-    * [Pull request 201](https://bitbucket.org/osrf/gazebo/pull-request/201)
+    * [BitBucket pull request 201](https://osrf-migration.github.io/gazebo-gh-pages/#!/osrf/gazebo/pull-requests/201)
 
 1. Filesystem copy directory and parentpath
-    * [Pull request 200](https://bitbucket.org/osrf/gazebo/pull-request/200)
+    * [BitBucket pull request 200](https://osrf-migration.github.io/gazebo-gh-pages/#!/osrf/gazebo/pull-requests/200)
 
 ## Ignition Common 3.2.0 (2019-08-07)
 
 1. Allow ignLogInit to use an absolute path. Added a timeToIso that converts a given time to an ISO string. Some console timestamps were using `IGN_SYSTEM_TIME_NS()` and others `ignition::common::systemTimeISO()`. Switched all to use `ignition::common::systemTimeISO()`.
-    * [Pull request 203](https://bitbucket.org/osrf/gazebo/pull-request/203)
+    * [BitBucket pull request 203](https://osrf-migration.github.io/gazebo-gh-pages/#!/osrf/gazebo/pull-requests/203)
 
 1. Port ColladaLoader fix: use default value of 1 for stride parameter when unset.
-    * [Pull request 196](https://bitbucket.org/osrf/gazebo/pull-request/196)
+    * [BitBucket pull request 196](https://osrf-migration.github.io/gazebo-gh-pages/#!/osrf/gazebo/pull-requests/196)
```

Comment:
The original changelog linked to the wrong repo. These tickets are in ign-common, not gazebo.

```suggestion
    * [BitBucket pull request 196](https://osrf-migration.github.io/ignition-gh-pages/#!/ignitionrobotics/ign-common/pull-requests/196)
```